src/Router/utils/router.js

```diff
@@ -262,6 +262,22 @@ const init = config => {
       console.error(`[Router]: ${config.bootComponent} is not a valid boot component`)
     }
   }
+  config.routes.forEach(item => {
+    let path = item.path
+    // Pattern to identify the last path of the route
+    // It should start with "/:" + any word  and ends with "?"
+    // Example /player/:asset/:assetId?
+    const pattern = /\/:{1}\w+\?$/
```

Comment:
`\w` matches `[A-Za-z0-9_]` character class, whereas as far as I see there is no restriction about path names in the Router docs. For example, they can include hyphen `-` or and other characters from other languages. Also `{1}` is not required when we try to match single literal character. So I think the following regex might be better:

```js
const pattern = /\/:.*?\?$/u
```